File: tuesThurs_sectionB/week12_nov10-14/thursNov13/Person.java
public class Person implements Comparable<Person> {
    private String name;
    private int age;

    public Person(String n, int a) {
        name = n;
        age = a;
    }

    public String toString() {
        return String.format("Name: %s, age: %d%n", name, age);
    }

    public int compareTo(Person other) {
        if (name.compareTo(other.name) == 0) {
            if (age < other.age) return -1;
            else if (age > other.age) return 1;
            else return 0;
        }
        else {
            return name.compareTo(other.name);
        }
    }
}

File: monWed_sectionA/week14_dec1-5/monDec1/dec1/Person.cs
public class Person {
    private int age = 0;

    public string Name { get; set; }

    //want no negatives
    //not more than 130
    public int Age
    {
        get {
            return age;
        }
        set {
            if (age >= 0 && age <= 130) {
                age = value;
            }
        }
    }

    public Person(string n, int a) {
        Name = n;
        Age = a;
    }
}